Description: Upstream support for multiple languages is broken. Always use "en"
  as language.
Bug: http://code.google.com/p/feedvalidator/issues/detail?id=10
Bug-Debian: http://bugs.debian.org/588983
Forwarded: not-needed

--- a/feedvalidator/formatter/base.py
+++ b/feedvalidator/formatter/base.py
@@ -8,7 +8,8 @@
 
 from UserList import UserList
 import os
-LANGUAGE = os.environ.get('LANGUAGE', 'en')
+# FIXME: upstream support for i18n is broken; hardcode "en"
+LANGUAGE = 'en'
 lang = __import__('feedvalidator.i18n.%s' % LANGUAGE, globals(), locals(), LANGUAGE)
 
 from feedvalidator.logging import Info, Warning, Error
